docs/index.rst

```diff
@@ -1,45 +1,22 @@
-.. |h4| raw:: html
-
-   <h4>
-
-.. |h4end| raw:: html
-
-   </h4>
-
-.. |br| raw:: html
-
-   <br />
-
 Welcome to Rockcraft's documentation!
 ===========================================
 
+Rockcraft is a tool to create :ref:`rocks_explanation`, a new generation
```

Comment:
have the guidelines changed? We've used the previous format on request. And TBH, the use of bold text makes it more readable and structured IMO.

Reply:
I'll put the bold back in.